torchvision/prototype/datasets/datapipes.py

```diff
@@ -0,0 +1,20 @@
+"""This is a temporary module for datapipes that are needed now, but should be absorbed by torchdata in the future."""
+
+import random
+from typing import Any
+
+from torch.utils.data import IterDataPipe
+
+
+class RandomPicker(IterDataPipe):
```

Comment:
https://github.com/facebookexternal/torchdata/blob/7881a93c6f83aa5c6a129c2123a42cf23c7dddd9/torchdata/datapipes/iter/__init__.py#L29 might help

Reply:
I'll test when `torchdata` is public.